### PR TITLE
[bitnami/thanos] Added minio reference link in values.yaml

### LINKED ISSUE #9158: minio security context
### Name and Version

bitnami/minio 1.2.3

### What is the problem this feature will solve?

run minio instance on openshift without high scc

### What is the feature you are proposing to solve the problem?

add podSecurityContext and containerSecurityContext options to minio, just as rest of the pods have.

also add some global.podSecurityContext and global.containerSecurityContext to the helm chart,
which will apply the security context to all of the instances.

### What alternatives have you considered?

_No response_

### PR DESCRIPTION
**Pull request by zfrhv**
Signed-off-by: Zahar Pecherichny <zfrhv2010@gmail.com>

**Description of the change**

adding minio link in thanos helm chart

**Benefits**

when i was looking at thanos helm chart i thoutht that i couldnt configure security context, so now everyone can see the full list of the variables that they can configure

**Possible drawbacks**

it feels a little hardcoded.

**Applicable issues**

  - partly fixes #9158

**Additional information**

 i think the best way to do it, is to change the [readme-generator](https://github.com/bitnami-labs/readme-generator-for-helm) to automatically detect the repo url and automatically add it for each dependency helm chart.

for example if the thanos chart is depended on minio chart.
so the readme-generator should go over Chart.yaml, see minio dependency, and try to add it in the readme.md.
some something like this:
```
// in Chart.yaml
for (const chart of dependencies) {
  exec(`helm repo add temp ${chart.repository}`)
  const home_url = exec(`helm show chart temp/${chart.name} | jq .home`)
  inject_to_readme(find "chart.name" and put there "for more values refer to ${home_url}")
}
```
or maybe make another tool that will inject the "@extra ${chart.name} for more values refer [here](${home_url})" in values.yaml instead of readme.md

**Checklist**

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the values.yaml and added to the README.md using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/thanos/values.yaml [hunk 1]
```diff
@@ -3675,6 +3675,7 @@ volumePermissions:
     pullSecrets: []
 
 ## @section MinIO&reg; chart parameters
+## @extra minio For full list of minio values configurations please regere [here](https://github.com/bitnami/charts/tree/master/bitnami/minio)
```
Comment (carrodher) [inline bitnami/thanos/values.yaml:3678]:
```suggestion
## @extra minio For full list of MinIO&reg; values configurations please refere [here](https://github.com/bitnami/charts/tree/master/bitnami/minio)
```

**Review by carrodher — COMMENTED** [review]
Thanks for your contribution, can you please take a look at the minor suggestion?

File: bitnami/thanos/values.yaml (hunk 1 above)
Comment (zfrhv) [reply bitnami/thanos/values.yaml]: sure, sorry my bad

**Review by carrodher — APPROVED** [review]
Thanks!

**Pull request merged**